homeassistant/components/neato/strings.json

```diff
@@ -12,14 +12,14 @@
       }
     },
     "error": {
-      "invalid_credentials": "Invalid credentials",
-      "unexpected_error": "Unexpected error"
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "create_entry": {
       "default": "See [Neato documentation]({docs_url})."
     },
     "abort": {
-      "already_configured": "Already configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "invalid_credentials": "Invalid credentials"
```

Comment:
Should we use the same key here as well since we replaced it up above?  I know we dont have an actual `abort` key for this so not 100% certain if we can change it here.

Reply:
Yes, we should change here too.

Reply:
I didn't find a good match to replace this with a reference, what do you suggest?

Reply:
@MBlokhuijzen `invalid_credentials` here is the same as above in the `error` that you had changed so it should also be `invalid_auth`

Reply:
Oke so the abort.invalid_credentials should be changed tot the key of invalid_auth and reference. I'll do that too